cmd/mungedocs/longCmd.go

```diff
@@ -0,0 +1,155 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+//split long commands in docs/ and examples
+package main
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+)
+
+var cmdRE = regexp.MustCompile(`^\s*(\$.*$)`)
```

Comment:
I can try to catch `kubectl` commands that miss the ^$. It's too hard to come up with a generic solution.